deps/v8/include/v8.h

```diff
@@ -5574,6 +6169,11 @@ class V8_EXPORT EmbedderHeapTracer {
    */
   virtual void AbortTracing() {}
 
+  /**
+   * Returns the number of wrappers that are still to be traced by the embedder.
+   */
+  virtual size_t NumberOfWrappersToTrace() { return 0; }
```

Comment:
Just for my own understanding – it doesn’t make sense for addons to set an `EmbedderHeapTracer`, which is why we don’t care about these breakages, right?